### PR TITLE
Show Transcription task instructions and add Assigned By column

### PR DESCRIPTION
**Pull request by libnava**
Resolves [CASEFLOW-2593](https://vajira.max.gov/browse/CASEFLOW-2593)

### Description
This adds a tooltip to Transcription tasks in the "Tasks" column of the Transcription Team's Unassigned tab table. It also adds an "Assigned By" column to that table, as we learned that would be helpful for the way they work their cases. 

### Testing Plan
1. Log in as the TRANSCRIPTION_USER
2. Open Queue
3. Switch your view to "Transcription team cases." 
4. If you do not see any transcription tasks, you may need to create one in the Rails console, and give it these values:
`assigned_to_type = "Organization"`
`assigned_to_id = Organization.find_by_name("Transcription").id`
`parent = AssignHearingDispositionTask.last  (and from the result of that, copy the appeal_id)`
`appeal = Appeal.find(copied_appeal_id)`
`instructions = ["Sample instruction"]`
5. Save it, and refresh the table. 

### User Facing Changes
TranscriptionTaskTooltip

### CODE REVIEW

File: client/app/queue/components/TaskTableColumns.jsx [hunk 1]
```diff
@@ -121,12 +122,35 @@ export const taskColumn = (tasks, filterOptions) => {
     filterOptions,
     label: 'Filter by task',
     valueName: 'label',
-    valueFunction: (task) => task.label,
+    valueFunction: (task) => {
+      if (task.label !== 'Transcription Task') {
+        return task.label;
+      }
+
+      const instructions = Array.isArray(task.instructions) ? task.instructions : [task.instructions];
```
Comment (nweinmeister) [inline client/app/queue/components/TaskTableColumns.jsx:130]: I think we can expect instructions to always be an array, but if we want to keep this check in place (might be better safe), I think it'd make more sense in the serializer as opposed to here. Thoughts?

File: client/app/queue/components/TaskTableColumns.jsx [hunk 2]
```diff
@@ -121,12 +122,35 @@ export const taskColumn = (tasks, filterOptions) => {
     filterOptions,
     label: 'Filter by task',
     valueName: 'label',
-    valueFunction: (task) => task.label,
+    valueFunction: (task) => {
+      if (task.label !== 'Transcription Task') {
```
Comment (nweinmeister) [inline client/app/queue/components/TaskTableColumns.jsx:126]: Can we pull 'Transcription Task' into a constant?

Comment (libnava) [reply client/app/queue/components/TaskTableColumns.jsx]: Good call, updated

File: client/app/queue/components/TaskTableColumns.jsx (hunk 1 above)
Comment (libnava) [reply client/app/queue/components/TaskTableColumns.jsx]: Moved to serializer, but yeah, not sure it's necessary overall.

**Review by nweinmeister — APPROVED** [review]
LGTM, will want to get approval from an echo team member before merge 👍

**Review by besampson0 — APPROVED** [review]
LGTM

**Pull request merged**